python/speeding_penalty.py

```diff
@@ -0,0 +1,3 @@
+def penalty(numbers):
+    numbers.sort(key=lambda x: x*2)
```

Comment:
Explain to the what `key=` is doing here and why it's needed.

Reply:
The key= allows adjustment to how the .sort() functions, in this example I use the a lamba function to x two times so that the string is more than one digit long, here is an example, the case of the input [32, 3], if you normally sort this list and then join it you get 332, however if you repeat each number twice you get 3232 and 33, which when sorted lexcicograplically, 3232 comes first which results in 323 which is smaller than 332, if you dont repeat each number twice you run into issues doing a lexicographical sort, in this case you end up with 3 coming before 32 which makes you end up with 332. Note:(the input numbers must be strings or be converted to strings to be sorted as such, as this all relys on the fact that the sort function sorts all strings lexicographically and not numerically)